.github/workflows/deploy-docs.yml

```
@@ -23,175 +21,67 @@
+            canary.c15t.com
+            preview-${{ github.ref_name }}.c15t.dev
+            pr-${{ github.event.pull_request.number }}.c15t.dev
+
+          # Orchestration & gating
+          consent_git_token: ${{ secrets.CONSENT_GIT_TOKEN }}
+          only_if_changed: "true"
+          change_globs: |
+            docs/**
+            packages/*/src/**
+            packages/*/package.json
+          check_template_changes: "true"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)